src/v0.8/interfaces/module/IDatasetsChallenge.sol

```diff
@@ -68,6 +68,14 @@ interface IDatasetsChallenge {
         uint64 _randomSeed
     ) external view returns (bool);
 
+    /// @notice Checks if the dataset audit has timed out.
+    /// @dev This function determines if the dataset audit for the given dataset ID has timed out.
+    /// @param _datasetId The ID of the dataset.
+    /// @return True if the dataset audit has timed out, false otherwise.
+    function isDatasetAuditTimeout(
```

Comment:
Need to add active trigger interface to switch status

Reply:
add reportDatasetWorkflowTimeout